addons/mail/static/src/components/discuss/discuss.scss

```diff
@@ -1,92 +1,8 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_Discuss {
-    display: flex;
-    height: map-get($sizes, 100);
     min-height: 0;
-
-    &.o-isDeviceSmall {
-        flex-flow: column;
-        align-items: center;
-    }
 }
 
 .o_Discuss_chatWindowHeader {
     width: map-get($sizes, 100);
     flex: 0 0 auto;
 }
-
-.o_Discuss_content {
-    height: map-get($sizes, 100);
-    overflow: auto;
-    flex: 1 1 auto;
-    display: flex;
-    flex-flow: column;
-}
-
-.o_Discuss_mobileAddItemHeader {
-    display: flex;
-    justify-content: center;
-    width: map-get($sizes, 100);
-    padding: map-get($spacers, 3);
-}
-
-.o_Discuss_mobileAddItemHeaderInput {
-    flex: 1 1 auto;
-    margin-bottom: map-get($spacers, 3);
-    padding: map-get($spacers, 2);
-}
-
-.o_Discuss_mobileMailboxSelection {
-    width: map-get($sizes, 100);
-}
-
-.o_Discuss_mobileNavbar {
-    width: map-get($sizes, 100);
-}
-
-.o_Discuss_noThread {
-    display: flex;
-    flex: 1 1 auto;
-    width: map-get($sizes, 100);
-    align-items: center;
-    justify-content: center;
-}
-
-.o_Discuss_sidebar {
-    height: map-get($sizes, 100);
```

Comment:
I see no `height` in classname (1st commit)

Reply:
Fixed 👍